src/bindgen/config.rs

```diff
@@ -669,12 +669,6 @@ impl EnumConfig {
         }
         self.private_default_tagged_enum_constructor
     }
-    pub(crate) fn swift_enum_macro(&self, annotations: &AnnotationSet) -> Option<String> {
```

Comment:
To be clear, I wasn't particularly against the annotation (though it probably isn't terribly useful and we can add it back if ever needed). I just thought that storing the "resolved" value shouldn't be necessary, since we store the annotations already.